### ISSUE TITLE
auth0: Error parsing JWK from url

### ISSUE THREAD

**Issue by tirumaraiselvan** [hasura/graphql-engine#5829]
Few auth0 projects are throwing this error.

**Comment by OmkarPh** [comment]
Hi @tirumaraiselvan 
I just came across this issue, I would like to work on it, but am understanding the repo setup now

**Comment by tiagob** [comment]
**tldr; https://community.auth0.com/t/certificate-thumbprint-is-longer-than-20-bytes/7794**

I came across this too. I've been using:

```
HASURA_GRAPHQL_JWT_SECRET={"jwk_url":"https://MY_TENANT.auth0.com/.well-known/jwks.json","audience":"hasura"}
```

This has generally been working fine but for one tenant it triggered the `Error parsing JWK from url`.

Switching to using https://hasura.io/jwt-config/ like the [docs](https://hasura.io/docs/1.0/graphql/core/guides/integrations/auth0-jwt.html#configure-hasura-to-use-auth0-keys) suggest fixed this.

```
HASURA_GRAPHQL_JWT_SECRET={"type": "RS512", "key": "-----BEGIN CERTIFICATE-----\nMY_CERTIFICATE\n-----END CERTIFICATE-----"}
```

I realized this is an old Auth0 tenant and the returned xt5 in the JWK is too long (40 characters instead of 20) so the other approach  is to [rotate the signing key](https://community.auth0.com/t/certificate-thumbprint-is-longer-than-20-bytes/7794) which should fix it
